.github/workflows/ci.yml

```
@@ -0,0 +1,254 @@
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20.9.0'
+
+      - name: Get cache key
+        id: cache-key
+        run: echo "value=${{ runner.os }}-node-${{ hashFiles('**/package.json', '**/yarn.lock') }}" >> $GITHUB_OUTPUT
+
+      - name: Cache dependencies
+        uses: actions/cache@v3
```

actionlint (1.7.7)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)